pom.xml

```diff
@@ -70,6 +70,12 @@ SOFTWARE.
     <developerConnection>scm:git:ssh://github.com:artipie/ppom.git</developerConnection>
     <url>https://github.com/artipie/ppom/tree/master</url>
   </scm>
+  <repositories>
```

Comment:
@Sammers21 I don't think it's a good decision to add snapshot repository for all adapters. It can be used as a workaround in artipie/asto for now, but we don't need to propagate snapshots dependencies on global level.

Reply:
asto has a transitive dependency: vertx of version 4.0.0-SNAPSHOT. They will not able to download it without the repository